packages/component-submission/server/use-cases/manuscript.js

```diff
@@ -77,7 +77,11 @@ class Manuscript {
   }
 
   async update(data) {
-    const manuscript = await ManuscriptModel.find(data.id, this.userId)
+    const manuscript = await ManuscriptModel.find(
```

Comment:
Personally, I'd like to see a comment to explain that we are just going to update `teams` as files are handled by the upload/remove functionality.